[
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.6.18",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2019-10156",
      "https://github.com/advisories/GHSA-grgm-pph5-j5h7"
    ],
    "url": "https://github.com/advisories/GHSA-grgm-pph5-j5h7"
  },
  {
    "severity": "CRITICAL",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.5.15",
    "source": "github",
    "summary": "Critical severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2019-3828",
      "https://github.com/advisories/GHSA-74vq-h4q8-x6jv"
    ],
    "url": "https://github.com/advisories/GHSA-74vq-h4q8-x6jv"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.6.1",
    "source": "github",
    "summary": "High severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-3096",
      "https://github.com/advisories/GHSA-rh6x-qvg7-rrmj"
    ],
    "url": "https://github.com/advisories/GHSA-rh6x-qvg7-rrmj"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.2",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2015-3908",
      "https://github.com/advisories/GHSA-w64c-pxjj-h866"
    ],
    "url": "https://github.com/advisories/GHSA-w64c-pxjj-h866"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.1.4",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-9587",
      "https://github.com/advisories/GHSA-m956-frf4-m2wr"
    ],
    "url": "https://github.com/advisories/GHSA-m956-frf4-m2wr"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.2.0",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-8628",
      "https://github.com/advisories/GHSA-jg4f-jqm5-4mgq"
    ],
    "url": "https://github.com/advisories/GHSA-jg4f-jqm5-4mgq"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.2.0",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-8614",
      "https://github.com/advisories/GHSA-cmwx-9m2h-x7v4"
    ],
    "url": "https://github.com/advisories/GHSA-cmwx-9m2h-x7v4"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.2.1.0",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-8647",
      "https://github.com/advisories/GHSA-x4cm-m36h-c6qj"
    ],
    "url": "https://github.com/advisories/GHSA-x4cm-m36h-c6qj"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.2.1",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2013-2233",
      "https://github.com/advisories/GHSA-9x6q-5423-w5v9"
    ],
    "url": "https://github.com/advisories/GHSA-9x6q-5423-w5v9"
  },
  {
    "severity": "MODERATE",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.3.1.0",
    "source": "github",
    "summary": "Moderate severity vulnerability that affects ansible",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2017-7481",
      "https://github.com/advisories/GHSA-w578-j992-554x"
    ],
    "url": "https://github.com/advisories/GHSA-w578-j992-554x"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.2.3",
    "source": "pyup",
    "summary": "ansible 1.2.3 includes local security fixes for predictable file locations for ControlPersist and retry file paths on shared machines on operating systems without kernel symlink/hardlink protections.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25616"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.4",
    "source": "pyup",
    "summary": "ansible 1.5.4 includes a security fix for safe_eval, which further hardens the checking of the evaluation function.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25617"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.5",
    "source": "pyup",
    "summary": "ansible 1.5.5 includes a security fix for vault, to ensure the umask is set to a restrictive mode before creating/editing vault files.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25618"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.6.4",
    "source": "pyup",
    "summary": "ansible includes 1.6.4 security updates related to evaluation of untrusted remote inputs.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25619"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.6.6",
    "source": "pyup",
    "summary": "ansible 1.6.6 includes security updates to further protect against the incorrect execution of untrusted data.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25620"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.6.7",
    "source": "pyup",
    "summary": "ansible 1.6.7 contains two security fixes:\r\n  * Strip lookup calls out of inventory variables and clean unsafe data\r\n    returned from lookup plugins (CVE-2014-4966)\r\n  * Make sure vars don't insert extra parameters into module args and prevent\r\n    duplicate params from superseding previous params (CVE-2014-4967)",
    "references": [],
    "url": "https://pyup.io/pyup.io-25621"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.7",
    "source": "pyup",
    "summary": "ansible 1.7 contains two security fixes:\r\n- Prevent the use of lookups when using legacy \" \" syntax around variables and with_* loops.\r\n  - Remove relative paths in TAR-archived file names used by ansible-galaxy.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25622"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.7.1",
    "source": "pyup",
    "summary": "ansible 1.7.1 contains a security fix to disallow specifying 'args:' as a string, which could allow the insertion of extra module parameters through variables.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25623"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.8.3",
    "source": "pyup",
    "summary": "ansible 1.8.3 fixes a security bug related to the default permissions set on a temporary file created when using \"ansible-vault view <filename>\".",
    "references": [],
    "url": "https://pyup.io/pyup.io-25624"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.2",
    "source": "pyup",
    "summary": "Ansible before 1.9.2 does not verify that the server hostname matches a domain name in the subject's Common Name (CN) or subjectAltName field of the X.509 certificate, which allows man-in-the-middle attackers to spoof SSL servers via an arbitrary valid certificate.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25625"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.6",
    "source": "pyup",
    "summary": "The create_script function in the lxc_container module in Ansible before 1.9.6-1 and 2.x before 2.0.2.0 allows local users to write to arbitrary files or gain privileges via a symlink attack on (1) /opt/.lxc-attach-script, (2) the archived container in the archive_path directory, or the (3) lxc-attach-script.log or (4) lxc-attach-script.err files in the temporary directory.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25626"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.0.2",
    "source": "pyup",
    "summary": "The create_script function in the lxc_container module in Ansible before 1.9.6-1 and 2.x before 2.0.2.0 allows local users to write to arbitrary files or gain privileges via a symlink attack on (1) /opt/.lxc-attach-script, (2) the archived container in the archive_path directory, or the (3) lxc-attach-script.log or (4) lxc-attach-script.err files in the temporary directory.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25627"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.2.1",
    "source": "pyup",
    "summary": "ansible before 2.2.1 is vulnerable to arbitrary code execution. An attacker with control over a client system being managed by Ansible and the ability to send facts back to the Ansible server could use this flaw to execute arbitrary code on the Ansible server as the user and group Ansible is running as.",
    "references": [],
    "url": "https://pyup.io/pyup.io-33286"
  },
  {
    "severity": "UNKNOWN",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.3.1",
    "source": "pyup",
    "summary": "ansible before 2.3.1 is vulnerable to CVE-2017-7481 - data for lookup plugins used as variables was not being correctly marked as \"unsafe\".",
    "references": [],
    "url": "https://pyup.io/pyup.io-34941"
  },
  {
    "severity": "UNKNOWN",
    "name": "archmage",
    "version": "0.3.0",
    "versions": "<0.3.1",
    "source": "pyup",
    "summary": "Directory traversal vulnerability in arCHMage 0.2.4 allows remote attackers to write to arbitrary files via a .. (dot dot) in a CHM file.",
    "references": [],
    "url": "https://pyup.io/pyup.io-25630"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.4",
    "source": "gemnasium",
    "summary": "Always-Incorrect Control Flow Implementation. Ansible mishandles the evaluation of some strings.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-2686"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-2686"
  },
  {
    "severity": "CRITICAL",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.4",
    "source": "gemnasium",
    "summary": "Security fix for safe_eval. The `safe_eval` function in Ansible does not properly restrict the code subset, which allows remote attackers to execute arbitrary code via crafted instructions.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-4657",
      "https://www.securityfocus.com/bid/68232",
      "https://github.com/ansible/ansible/blob/release1.5.5/CHANGELOG.md",
      "https://github.com/ansible/ansible/commit/998793fd0ab55705d57527a38cee5e83f535974c"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-4657"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.5",
    "source": "gemnasium",
    "summary": "Security fix for vault. The vault subsystem in Ansible does not set the umask before creation or modification of a vault file, which allows local users to obtain sensitive key information by reading a file.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-4658",
      "https://www.securityfocus.com/bid/68233",
      "https://github.com/ansible/ansible/blob/release1.5.5/CHANGELOG.md",
      "https://github.com/ansible/ansible/commit/a0e027fe362fbc209dbeff2f72d6e95f39885c69"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-4658"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.5",
    "source": "gemnasium",
    "summary": "Unsafe file permissions. Ansible sets 0644 permissions for `sources.list`, which might allow local users to obtain sensitive credential information in opportunistic circumstances by reading a file that uses the `deb http://user:pass@server:port/` format.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-4659",
      "https://www.securityfocus.com/bid/68234",
      "https://github.com/ansible/ansible/blob/release1.5.5/CHANGELOG.md",
      "https://github.com/ansible/ansible/commit/c4b5e46054c74176b2446c82d4df1a2610eddc08"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-4659"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.5.5",
    "source": "gemnasium",
    "summary": "Insufficiently Protected Credentials. Ansible constructs filenames containing user and password fields on the basis of deb lines in `sources.list`, which might allow local users to obtain sensitive credential information in opportunistic circumstances by leveraging existence of a file that uses the `deb http://user:pass@server:port/` format.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-4660",
      "https://www.securityfocus.com/bid/68231"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-4660"
  },
  {
    "severity": "CRITICAL",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.6.4",
    "source": "gemnasium",
    "summary": "Security fix for safe_eval. The `safe_eval` function in Ansible does not properly restrict the code subset, which allows remote attackers to execute arbitrary code via crafted instructions.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2014-4678",
      "https://github.com/ansible/ansible/commit/5429b85b9f6c2e640074176f36ff05fd5e4d1916",
      "https://groups.google.com/forum/message/raw?msg=ansible-announce/ieV1vZvcTXU/5Q93ThkY9rIJ"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-4678"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.2",
    "source": "gemnasium",
    "summary": "Improper TLS Certificate Validation. Ansible fails to adequately validate HTTPS certificates when using the get_url and uri modules, and when using the url and etcd lookup plugins. This allows for man-in-the-middle attacks on those connections.",
    "references": [
      "http://www.ansible.com/security",
      "https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#192-dancing-in-the-street---jun-26-2015"
    ],
    "url": "http://www.ansible.com/security"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<1.9.2",
    "source": "gemnasium",
    "summary": "Possible escape from confined environment. Improprer symlink handling in zone, jail, and chroot connection plugins could lead to escape from confined environment. File copying is not performed by code that is operating within the restricted environment (chroot, jail, or zone).",
    "references": [
      "http://www.ansible.com/security",
      "https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#192-dancing-in-the-street---jun-26-2015",
      "https://github.com/ansible/ansible/commit/952166f48eb0f5797b75b160fd156bbe1e8fc647",
      "https://github.com/ansible/ansible/commit/ca2f2c4ebd7b5e097eab0a710f79c1f63badf95b"
    ],
    "url": "http://www.ansible.com/security"
  },
  {
    "severity": "LOW",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.5.12,>=2.6.0.dev0,<2.6.9,>=2.7.0.dev0,<2.7.3",
    "source": "gemnasium",
    "summary": "Information Exposure. Execution of Ansible playbooks on Windows platforms with PowerShell ScriptBlock logging and Module logging enabled can allow for 'become' passwords to appear in EventLogs in plaintext. A local user with administrator privileges on the machine can view these logs and discover the plaintext password.",
    "references": [
      "http://www.securityfocus.com/bid/106004",
      "https://access.redhat.com/errata/RHSA-2018:3770",
      "https://access.redhat.com/errata/RHSA-2018:3771",
      "https://access.redhat.com/errata/RHSA-2018:3772",
      "https://access.redhat.com/errata/RHSA-2018:3773",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-16859",
      "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16859",
      "https://cwe.mitre.org/data/definitions/200.html",
      "https://github.com/ansible/ansible/commit/8c1f701e6e9df29fe991f98265e2dd76acca4b8c",
      "https://github.com/ansible/ansible/pull/49142"
    ],
    "url": "http://www.securityfocus.com/bid/106004"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.5.14,>=2.6.0.dev0,<2.6.11,>=2.7.0.dev0,<2.7.5",
    "source": "gemnasium",
    "summary": "Information Exposure. ansible is vulnerable to an information disclosure flaw in vvv+ mode with no_log on that can lead to leakage of sensible data.",
    "references": [
      "http://www.securityfocus.com/bid/106225",
      "https://access.redhat.com/errata/RHSA-2018:3835",
      "https://access.redhat.com/errata/RHSA-2018:3836",
      "https://access.redhat.com/errata/RHSA-2018:3837",
      "https://access.redhat.com/errata/RHSA-2018:3838",
      "https://access.redhat.com/errata/RHSA-2019:0564",
      "https://access.redhat.com/errata/RHSA-2019:0590",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-16876",
      "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16876",
      "https://cwe.mitre.org/data/definitions/200.html",
      "https://github.com/ansible/ansible/pull/49569",
      "https://www.debian.org/security/2019/dsa-4396"
    ],
    "url": "http://www.securityfocus.com/bid/106225"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.6.18,>=2.7.0,<2.7.12,>=2.8.0,<2.8.2",
    "source": "gemnasium",
    "summary": "Information Exposure. A flaw was discovered in the way Ansible templating was implemented causing the possibility of information disclosure through unexpected variable substitution. By taking advantage of unintended variable substitution the content of any variable may be disclosed.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2019-10156",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-10156"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-10156"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Exposure of Resource to Wrong Sphere. A race condition flaw was found in Ansible Engine when running a playbook with an unprivileged become user. When Ansible needs to run a module with become user, the temporary directory is created in `/var/tmp`. This directory is created with `umask 77 && mkdir -p <dir>`; this operation does not fail if the directory already exists and is owned by another user. An attacker could take advantage to gain control of the become user as the target directory can be retrieved by iterating `/proc/<pid>/cmdline`.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1733",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1733"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,2.8.8,2.9.5",
    "source": "gemnasium",
    "summary": "OS Command Injection. A flaw was found in the pipe lookup plugin of ansible. Arbitrary commands can be run, when the pipe lookup plugin uses `subprocess.Popen()` with `shell=True`, by overwriting ansible facts and the variable is not escaped by quote plugin. An attacker could take advantage and run arbitrary commands by overwriting the ansible facts.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1734",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1734"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1734"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Path Traversal. A flaw was found in the Ansible Engine when the fetch module is used. An attacker could intercept the module, inject a new path, and then choose a new destination path on the controller node.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1735",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1735"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1735"
  },
  {
    "severity": "LOW",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Incorrect Permission Assignment for Critical Resource. A flaw was found in Ansible Engine when a file is moved using `atomic_move` primitive as the file mode cannot be specified. This sets the destination files world-readable if the destination file does not exist and if the file exists, the file could be changed to have less restrictive permissions before the move. This could lead to the disclosure of sensitive data.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1736",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1736"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1736"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<2.7.17,>=2.8.0,<2.8.9,>=2.9.0,<2.9.6",
    "source": "gemnasium",
    "summary": "Path Traversal. When using the Extract-Zip function from the `win_unzip` module, extracted file(s) are not checked if they belong to the destination folder. An attacker could take advantage of this flaw by crafting an archive anywhere in the file system, using a path traversal.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1737",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1737"
  },
  {
    "severity": "HIGH",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Argument Injection or Modification. A flaw was found in Ansible Engine when the module package or service is used and the parameter `use` is not specified. If a previous task is executed with a malicious user, the module sent can be selected by the attacker using the ansible facts file.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1738",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1738"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1738"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Information Exposure. When a password is set with the argument `password` of svn module, it is used on svn command line, disclosing to other users within the same node. An attacker could take advantage by reading the cmdline file from that particular PID on the procfs.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1739",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739",
      "https://github.com/ansible/ansible/issues/67797"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1739"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Information Exposure. A flaw was found in Ansible Engine when using Ansible Vault for editing encrypted files. When a user executes `ansible-vault edit`, another user on the same computer can read the old and new secret, as it is created in a temporary file with mkstemp and the returned file descriptor is closed and the method write_data is called to write the existing secret in the file. This method will delete the file before recreating it insecurely.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1740",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1740"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1740"
  },
  {
    "severity": "MEDIUM",
    "name": "ansible",
    "version": "1.2.0",
    "versions": "<=2.7.16,>=2.8.0,<=2.8.8,>=2.9.0,<=2.9.5",
    "source": "gemnasium",
    "summary": "Inclusion of Sensitive Information in Log Files. Sensitive parameters such as passwords and tokens are passed to `kubectl` from the command line, not using an environment variable or an input configuration file. This will disclose passwords and tokens from process list and no_log directive from debug module would not have any effect making these secrets being disclosed on stdout and log files.",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2020-1753",
      "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1753"
    ],
    "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-1753"
  }
]
